shared/profile/generic/proofs-list/container.js

```diff
@@ -20,18 +20,31 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => ({
   onCancel: dispatchProps.onCancel,
   onClickLearn: () => openURL('https://keybase.io/docs/proof_integration_guide'),
   providerClicked: dispatchProps.providerClicked,
-  providers: stateProps._proofSuggestions
+  providers: promoteNew(stateProps._proofSuggestions
```

Comment:
Could consider doing something like:
```js
import {flatten, partition} from 'lodash-es'

providers: flatten(partition(stateProps._proofSuggestions, 'new'))
```

Reply:
partition returns a tuple and flatten takes an array. If only flatten took a $ReadOnlyArray